File: FlightManagementIntegrated/src/main/java/com/capg/flightMgmtSystem/entities/BookingDetails.java
package com.capg.flightMgmtSystem.entities;

import java.util.Date;

import javax.persistence.CascadeType;
import javax.persistence.Column;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@EqualsAndHashCode
@Getter
@Setter
public class BookingDetails {
	private double ticketCost;
	private int numberOfPassengers;
	private User user;
	private Passenger passenger;
	private ScheduledFlight scheduledFlight;
	
	

}

File: FlightMgmtSystem/src/app/BookingDetails.ts
import { Passenger } from './Passenger';
import { ScheduledFlight } from './ScheduledFlight';
import { User } from './User';

export class BookingDetails{
	 bookingDate:Date;
	 ticketCost:number;
	 numberOfPassengers:number;
	 user:User;
	 passenger:Passenger;
	 scheduledFlight:ScheduledFlight;

}

